packages/graphql-transformers-e2e-tests/src/__tests__/SubscriptionsRuntimeFiltering.e2e.test.ts

```diff
@@ -505,17 +514,36 @@ test('Multiple owners auth is supported for subscriptions', async () => {
   }) as unknown as Observable<any>;
   let subscription: ZenObservable.Subscription;
   const subscriptionPromise = new Promise((resolve, _) => {
-    subscription = failedObserver.subscribe(
-      event => {},
-      err => {
-        expect(err.error.errors[0].message).toEqual(
-          'Connection failed: {"errors":[{"errorType":"Unauthorized","message":"Not Authorized to access onCreateTaskGroup on type TaskGroup"}]}',
-        );
+    subscription = observer.subscribe(
```

Comment:
nit: We can also add a test for non-list type dynamic auth since we use different filters (`in` or `containsAny`)